apps/web/src/components/Settings/Profile/Picture.tsx

```diff
@@ -128,7 +115,8 @@ const Picture: FC<PictureProps> = ({ profile }) => {
 
     try {
       setUploading(true);
-      let { ipfsUrl, dataUrl } = await uploadImage(croppedImage);
+      const ipfsUrl = await uploadCroppedImage(croppedImage);
+      const dataUrl = croppedImage.toDataURL('image/png');
```

Comment:
Are we converting any type of image to png?

Reply:
Yes, but actually it seems that the backend converts everything to WebP format when it's uploaded, (if you download an avatar image or a cover image, the format is WebP). So I guess that PNG is to be seen as an intermediate format here, for uploading. I have not experimented with different formats though.

Reply:
Makes sense 👍🏼